tools/mmp/driver.cs

```diff
@@ -771,12 +773,35 @@ static void Pack (IList<string> unprocessed)
 			if (App.LinkMode != LinkMode.All && App.RuntimeOptions != null)
 				App.RuntimeOptions.Write (App.AppDirectory);
 
+			if (experimental_xm_aot.HasValue && experimental_xm_aot.Value && IsUnified) {
```

Comment:
Maybe show an error if aot is requested for Classic instead of silently ignoring it.